src/GitVersionTask/AssemblyInfoBuilder/AssemblyInfoBuilder.cs

```diff
@@ -10,7 +10,17 @@ public string GetAssemblyInfoText(VersionVariables vars, string assemblyName)
         var v = vars.ToList();
 
         var assemblyInfo = string.Format(
-@"using System;
+@"//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     GitVersion
```

Comment:
:memo: I did not include the version of GitVersion on this line because it's not already part of the output. Many tools do include this information; it's something to consider especially for cases where this file will not be part of source control.